.golangci.yaml

```diff
@@ -57,13 +57,13 @@ linters:
     - rowserrcheck
     - sqlclosecheck
     - stylecheck
-    - tenv
```

Comment:
The `tenv` linter is deprecated, replaced by `usetesting` (which really goes into action with Kubernetes v1.24).